hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -841,24 +851,20 @@
     {
       name: chip_tap_straps_dev
       uvm_test_seq: chip_tap_straps_vseq
-      en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+use_otp_image=LcStDev",
-                 "+create_jtag_riscv_map=1",
-                 // select DFT tap
-                 "+uart0_sel=0",
-                 // test will exit while SW is still running
-                 "+disable_assert_final_checks"]
+      en_run_modes: ["strap_tests_mode"]
+      run_opts: ["+use_otp_image=LcStDev"]
     }
     {
       name: chip_tap_straps_rma
       uvm_test_seq: chip_tap_straps_vseq
-      en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+use_otp_image=LcStRma",
-                 "+create_jtag_riscv_map=1",
-                 // select DFT tap
-                 "+uart0_sel=0",
-                 // test will exit while SW is still running
-                 "+disable_assert_final_checks"]
+      en_run_modes: ["strap_tests_mode"]
+      run_opts: ["+use_otp_image=LcStRma"]
```

Comment:
Do we need this, I thought the default image is RMA?

Reply:
yeah, that's right. Let me clean this up in another PR. Thanks